.gitignore

```diff
@@ -22,6 +22,7 @@ DerivedData
 *.hmap
 *.ipa
 *.xcuserstate
+ios/.xcode.env.local
```

Comment:
Can we anchor both this and the following `ios/` entries to the root `/ios` directory?
```suggestion
/ios/.xcode.env.local
```

Reply:
Sure!